ocamltest/ocaml_tests.ml

```diff
@@ -20,51 +20,6 @@ open Builtin_actions
 open Ocaml_actions
 
 let bytecode =
-<<<<<<< HEAD
-  let test_actions =
-    if not Ocamltest_config.native_compiler then
-      [
-        setup_ocamlc_byte_build_env;
-        ocamlc_byte;
-        check_ocamlc_byte_output;
-        run;
-        check_program_output;
```

Comment:
In `tip-jst`, we would not run these actions when `Ocamltest_config.native_compiler` is true:

```
        setup_ocamlc_byte_build_env;
        ocamlc_byte;
        check_ocamlc_byte_output;
```
(And this seems like an intentional divergence from upstream; see https://github.com/ocaml-flambda/flambda-backend/pull/504.)

In the resolved code, we do run those actions when `Ocamltest_config.native_compiler` is true. Should we try to fix that here?

Reply:
It seems like an equivalent of PR504 (but not involving the changes to `ocaml_compilers.ml`) was already merged in 5.1 but enabled only on Windows.  I've now enabled it unconditionally to match PR504, and have put back in the change in `ocaml_compilers.ml` from that PR.

Reply:
Thanks